langchain/package.json

```diff
@@ -856,8 +856,8 @@
     "@faker-js/faker": "^7.6.0",
     "@getmetal/metal-sdk": "^4.0.0",
     "@getzep/zep-js": "^0.9.0",
-    "@gomomento/sdk": "^1.47.1",
-    "@gomomento/sdk-core": "^1.47.1",
+    "@gomomento/sdk": "^1.51.1",
+    "@gomomento/sdk-core": "^1.51.1",
```

Comment:
Bumped to get `searchAndFetchVectors` support in Momento client. `@gomomento/sdk` has the actual client implementation (needed for integration tests) and `sdk-core` for the interface and message types.